src/sentry/api/endpoints/organization_events_facets_performance.py

```diff
@@ -0,0 +1,50 @@
+import sentry_sdk
+
+from collections import defaultdict
+from rest_framework.response import Response
+
+from sentry.api.bases import OrganizationEventsV2EndpointBase, NoProjects
+from sentry.snuba import discover
+from sentry import features, tagstore
+
+
+class OrganizationEventsFacetsPerformanceEndpoint(OrganizationEventsV2EndpointBase):
+    def has_feature(self, organization, request):
+        return features.has(
+            "organizations:performance-tag-explorer", organization, actor=request.user
+        )
+
+    def get(self, request, organization):
+        if not self.has_feature(organization, request):
+            return Response(status=404)
+
+        try:
+            params = self.get_snuba_params(request, organization)
+        except NoProjects:
+            return Response([])
+
+        aggregate_column = request.GET.get("aggregateColumn", "duration")
```

Comment:
nit since this is an experiment, but we should validate this value in future iterations

Reply:
Yeah, mentioned it in description, but I think we can turn `aggregateColumn` and maybe `aggregateFunction` into validated params.